spark/sql-13/build.gradle

```diff
@@ -116,7 +124,9 @@ jar {
 }
 
 javadoc {
+    dependsOn scaladoc //ensures that scala compiles and scala docs are valid
```

Comment:
We don't need to run `scaladoc` to generate the javadocs. We just need to _compile_. This should depend on `compileScala` instead.

Reply:
since nothing else actually runs the scala docs, I thought that generating them next to the java docs (even if not used) would serve as a good as test. But it is unnecessary and I will change it.